File: Java/extra/Lista 01/src/ex04.java
// Lê dois números e mostra a soma. Ante do resultado, deverá aparecer a mensagem: SOMA

import java.util.Scanner;

public class ex04 {
    public static void main(String[] args) {
        int num1, num2;
        try (Scanner leitor = new Scanner(System.in)) {
            System.out.println("Digite o primeiro número: ");
            num1 = leitor.nextInt();
            System.out.println("\nDigite o segundo número: ");
            num2 = leitor.nextInt();
        }
        System.out.format("\nSOMA\n%d + %d = %d", num1, num2, soma(num1, num2));
    }

    static int soma(int a, int b) {
        return a + b;
    }
}

File: C/Listas/Lista 01/ex04.c
//Lê dois números e mostra a soma. Ante do resultado, deverá aparecer a mensagem: SOMA
#include <stdio.h>

int main(void){
    int n1, n2;
    printf("Digite o primeiro número: ");
    scanf("%d", &n1);
    printf("Digite o segundo número: ");
    scanf("%d", &n2);

    printf("\nSOMA\n%d + %d",n1,n2);
}